executor/aggregate.go

```diff
@@ -811,12 +811,14 @@ func (e *StreamAggExec) Close() error {
 func (e *StreamAggExec) Next(ctx context.Context, chk *chunk.Chunk) error {
 	chk.Reset()
 
-	for !e.executed && chk.NumRows() < e.maxChunkSize {
-		err := e.consumeOneGroup(ctx, chk)
-		if err != nil {
-			e.executed = true
-			return errors.Trace(err)
-		}
+	if e.executed || chk.NumRows() >= e.maxChunkSize {
```

Comment:
what does `chk.NumRows() >= e.maxChunkSize` mean?

Reply:
oh....I shouldn't modify that..- -